### DISCUSSION TITLE
"Unknown options: s3://***/" error when trying to upload to S3 through Github Actions

### DISCUSSION THREAD

**Discussion by aceesay94** [community/community#120138] [Actions]
Hey guys! I'm new to Github and programming overall. I am trying to complete the AWS Cloud Resume Challenge. I'm trying to upload to S3 using Github Actions, however even though I have the necessary secrets in my Github Actions and the seemingly the correct IAM permissions, I am consistently getting "Unknown options: s3://***/". If you have any idea what may be the problem, I would greatly appreciate it! This is how the code looks 
<img width="1127" alt="Screenshot 2024-04-18 at 2 42 59 PM" src="https://github.com/community/community/assets/160535936/40209b96-1ab7-46d4-bd71-84a1ab61ef92">

and this is the error message I keep receiving: 
<img width="1095" alt="Screenshot 2024-04-18 at 2 44 50 PM" src="https://github.com/community/community/assets/160535936/f21ea8ef-9b51-4464-81d2-9cf2692726a7">

**Comment by ebndev** [comment]
Welcome to the GitHub Community, @aceesay94, we're happy you're here! 

I've gone ahead and moved this to our Actions category for you to hopefully get you in the right direction. Good luck!
